tutorials/deploy-xenforo-to-compute-engine/index.md

```diff
@@ -0,0 +1,267 @@
+	* Allow HTTP and HTTPS traffic to your instance.
+	* Click Create. It takes a short while for Compute Engine to create and start your instance.
+
+	Depending on the scale of your website, you may need to add more CPU cores, increase the amount of memory available and use a large SSD persistent boot disk instead of the default 10GB standard one. Though for this tutorial it is OK to leave these settings unchanged. It is also possible to upgrade (or downgrade) your instance after deployment.
+
+	For instructions on creating and starting a Compute Engine instance via Cloud SDK or with Compute Engine API, refer to the [Creating and Starting an Instance page](https://cloud.google.com/compute/docs/instances/create-start-instance) in Compute Engine documentation.
+
+5. Download your copy of XenForo from XenForo’s website. For this instruction you need to use XenForo 1.5.
+
+## Understanding the Architecture
+
+XenForo needs to work with a web server software and a MySQL database backend to serve requests from the Internet. In this tutorial you will use [Apache HTTP server](https://httpd.apache.org/) for web server software and [MariaDB](https://mariadb.org/) for database backend. At a basic level, the architecture of the project looks as follows:
+
+![Understanding the Architecture](/tutorials/deploy-xenforo-to-compute-engine/architecture_overview.png)
+
+## Installing the Prerequisites
+
+You need to connect to your Compute Engine instance to complete the following steps. To connect from your browser, go to the [VM Instances page](https://console.cloud.google.com/compute/instances) and click **SSH** in the row of the instance that you want to connect to. For other ways of connecting to instances, refer to the [Connecting to Linux Instances](https://cloud.google.com/compute/docs/instances/connecting-to-instance) page of Compute Engine documentation.
+
+1. Update package lists to get information on the newest packages and their dependencies:
+
+	```bash
+	sudo apt-get update
+	```
+
+2. Install Apache server and its module for PHP support:
+
+	```bash
+	sudo apt-get install apache2 apache2-mod-php7.0 -y
+	```
+
+	After the installation is completed, open your browser and visit http://`[SERVER_EXTERNAL_IP]` (note that it is a HTTP link instead of a HTTPS link). `[SERVER_EXTERNAL_IP]` can be found on the [VM Instances](https://console.cloud.google.com/compute/instances) page, in the row of the instance that you connect to. You should see a page titled “Apache Debian Default Page” in your browser. 
+
+3. Install MariaDB server:
+
+	```bash
+	sudo apt-get install mariadb-server -y
+	```
+
+4. Set up MariaDB:
+
+	```bash
+	sudo mysql_secure_installation
+	```
+
+	The script prepares MariaDB for production use. For security reasons you should follow the prompts on screen, set up a root password, remove all anonymous users, disable remote login for root user, remove test database and reload the privilege tables.
```

Comment:
"For security reasons, you should follow the on-screen prompts, set up a root password, remove all anonymous users, disable remote login for the root user, remove test database, and reload the privilege tables."